Sources/RayTracer.Sample/Program.cs

```diff
@@ -1,19 +1,40 @@
 ﻿using System.IO;
 using RayTracer.Library.IO;
+using RayTracer.Library.Mathematics;
 using RayTracer.Library.Serialization;
+using RayTracer.Library.Shapes;
 using RayTracer.Library.Utils;
 
 CameraSettings settings = CameraSettings.Default with
 {
-    ImageHeight = 20
+    ImageHeight = 100
 };
 
 Camera camera = new(settings);
 
 FileStream fs = File.OpenRead("../../../../../Assets/Scenes/SpheresTest.json");
 Scene scene = SerializationHelper.Deserialize<Scene>(fs)!;
+//scene.Shapes.Add(new Plane(new Vector3(0,0,-3), Vector3.Normalize(new Vector3(0,0,1))));
```

Comment:
pls revert all changes to this file so it remains clean

Reply:
Done